scargo/file_generators/templates/.gitlab-ci.yml.j2

```diff
@@ -52,6 +52,9 @@ tests:
   extends: .merge_request
   allow_failure: true
   script:
+  {% if project.target.family == "esp32" %}
+    - source /opt/esp/entrypoint.sh
```

Comment:
This means that we end up with different `clang-format` versions for different targets. That's bound to cause problems down the line.

We should make sure that we use the same version across all targets (so we probably want to avoid using the one from esp-idf).

Reply:
Good point, I've changed the approach.